paperqa/llms.py

```diff
@@ -794,16 +798,36 @@ def add_texts_and_embeddings(self, texts: Iterable[Embeddable]) -> None:
 
     @abstractmethod
     async def similarity_search(
-        self, query: str, k: int, embedding_model: EmbeddingModel
+        self,
+        query: str,
+        k: int,
+        embedding_model: EmbeddingModel,
```

Comment:
```suggestion
        embedding_model: EmbeddingModel
```

Just avoiding diff churn